packages/strapi/lib/services/__tests__/entity-service.test.js

```diff
@@ -88,6 +88,7 @@ describe('Entity service', () => {
         const fakeModel = {
           kind: 'contentType',
           modelName: 'test-model',
+          options: {},
```

Comment:
Why do you need to add this here?

Reply:
Because the test was broken "options is undefined"